versionberg/src/main/groovy/io/github/rockerhieu/versionberg/Git.groovy

```diff
@@ -26,9 +26,12 @@ package io.github.rockerhieu.versionberg
  * Created by rockerhieu on 9/12/16.
  */
 class Git {
+    static String repositoryPath = ''
+
     static String getCommitSha() {
         try {
-            return 'git rev-parse --short HEAD'.execute().text.trim()
+            def externalPathOption = checkInnerGitRepo() ? "--git-dir=${repositoryPath}/.git" : ''
```

Comment:
`checkInnerGitRepo` is unnecessary as `getCommitSha` can just accept `gitDir` as a parameter

Reply:
To be clear, here you can always use `gitDir` in the git command.